components/bl-multiple-select-component/src/options/index.js

```diff
@@ -1,21 +1,23 @@
-import { useState, useEffect, useRef, useMemo, useCallback } from 'react';
+import { useState, useLayoutEffect, useRef, useMemo, useCallback } from 'react';
 
 import { Option } from './option';
 import { SelectAllCheckbox } from './select-all-checkbox';
 
+const { cn } = BackendlessUI.CSSUtils;
+
 const DEFAULT = 'default';
 
 export function Options(props) {
   const { type, options, selectValue, selectAllCheckbox, selectAllLabel, setSelectValue, onChange } = props;
   const selectRef = useRef(null);
-  const [margin, setMargin] = useState(0);
+  const [optionsPlacement, setOptionsPlacement] = useState('bottom');
```

Comment:
And what if change the state logic? Change `OptionsPlacement` to an `isOptionsOnTop`. In this case, will not need an `OptionPlacement` map. Because the main task of this state is to check if the options are on top.

Does this make sense?